wallet/src/main/java/com/android/identity/issuance/remote/StorageImpl.kt

```diff
@@ -58,6 +61,15 @@ internal class StorageImpl(
                 null,
                 null
             )
+            // TODO: Older OS versions don't support setting the cursor window size.
+            //  What should we do with older OS versions?
+            //  Also note that a large window size may lead to longer delays when loading from the
+            //  database. And if we keep this, replace the magic number with a constant.
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+                // The default window size of 2MB is too small for video files.
+                (cursor as? AbstractWindowedCursor)?.window = CursorWindow(
+                    "Larger Window", 256 * 1024 * 1024)
+            }
```

Comment:
I think it's probably fine for the Wallet to work on only Android 9 (P) or higher...